network/kademlia.go

```diff
@@ -307,12 +308,18 @@ func (k *Kademlia) SuggestPeer() (suggestedPeer *BzzAddr, saturationDepth int, c
 	// insert them in a map of bin arrays, keyed with the number of connected peers
 	saturation := make(map[int][]int)
 	var lastPO int       // the last non-empty PO bin in the iteration
-	saturationDepth = -1 // the deepest PO such that all shallower bins have >= k.MinBinSize peers
+	saturationDepth = -1 // the deepest PO such that all shallower bins have >= expectedMinBinSize peers
 	var pastDepth bool   // whether po of iteration >= depth
-	k.conns.EachBin(k.base, Pof, 0, func(bin *pot.Bin) bool {
-		// process skipped empty bins
+	var currentMaxMinBinSize = k.MinBinSize
```

Comment:
"MaxMin" ? :)

Reply:
current maximum minBinSize. Maybe rename it to "maximumMinBinSize"?

Reply:
or `MaxBinSize`?